var/spack/repos/builtin/packages/cuda/package.py

```diff
@@ -23,6 +23,7 @@
 
 _versions = {
     '11.0.2': {
+        'Linux-aarch64': ('3942f368a9afab99ced1df20168f02f5', 'http://developer.download.nvidia.com/compute/cuda/11.0.2/local_installers/cuda_11.0.2_450.51.05_linux_sbsa.run'),
```

Comment:
Can you please use the sha256 hash for consistency? :)

Reply:
Sure I will.

Reply:
```suggestion
        'Linux-aarch64': ('23851e30f7c47a1baad92891abde0adbc783de5962c7480b9725198ceacda4a0', 'http://developer.download.nvidia.com/compute/cuda/11.0.2/local_installers/cuda_11.0.2_450.51.05_linux_sbsa.run'),
```